client/src/main/java/io/spine/client/QueryFilter.java

```diff
@@ -36,11 +36,11 @@
 /**
  * A query filter which targets a {@linkplain EntityColumn column} of an entity.
  */
-public final class QueryFilter extends TypedFilter<EntityState> {
+public final class QueryFilter extends TypedFilter<EntityState<?>> {
```

Comment:
What if we have generic parameter `V` for this class? The static factories below do accept it. Why cannot we make it a bit more save on the level of the class declaration too?